### PR TITLE
restart command

### PR DESCRIPTION
**Pull request by JackCrispy**
dab

### CODE REVIEW

**Comment by Lireoy** [comment]
1. Code style should conform to standard.
2. The server invite command was discussed before in #730: 
image
4. Restart already exists in console with the help of [nodemon](https://nodemon.io/).

**Comment by JackCrispy** [comment]
yes, the restart command is in the console, this is a command to restart from discord, and the serverinvite file was deleted.

**Comment by Lireoy** [comment]
If the server invite command was removed, please remove the related strings from `commands.json`.

File: src/commands/developers/restart.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+    this.requirements = new CommandRequirements(this, { devOnly: true })
+  }
+
+  async run({ message }) {
+    const embed = new SwitchbladeEmbed();
+
+    message.channel.send(embed
+      .setTitle('Service Restart')
+      .setDescription('Restarting...')
```
Comment (Lireoy) [inline src/commands/developers/restart.js]: Also, no hard coded strings please.

**Comment by pedrofracassi** [comment]
Why would we need to restart from discord? Also that's not a full restart. You're just deleting the client and creating it again

File: src/commands/developers/restart.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+const { CommandStructures, SwitchbladeEmbed } = require('../../')
+const { Command, CommandRequirements } = CommandStructures
+
+module.exports = class Restart extends Command {
+  constructor(client) {
+    super(client)
+    this.name = 'restart'
```
Comment (Doges) [inline src/commands/developers/restart.js]: we changed our system

File: src/commands/developers/restart.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    this.requirements = new CommandRequirements(this, { devOnly: true })
+  }
+
+  async run({ message }) {
+    const embed = new SwitchbladeEmbed();
+
+    message.channel.send(embed
+      .setTitle('Service Restart')
+      .setDescription('Restarting...')
+    ).then(message.client.destroy()).then(message.client.login());
```
Comment (metehus) [inline src/commands/developers/restart.js]: This wont work, you need to use `.then(() => message.client.destroy())` and please no `;`. fucking run `npm run test`

File: src/commands/developers/restart.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    this.category = 'developers'
+    this.hidden = true
+
+    this.requirements = new CommandRequirements(this, { devOnly: true })
+  }
+
+  async run({ message }) {
+    const embed = new SwitchbladeEmbed();
+
+    message.channel.send(embed
```
Comment (metehus) [inline src/commands/developers/restart.js]: you could just use channel by only calling the channel from the command context

**Comment by metehus** [comment]
anyways why a restart command?

**Pull request closed without merging**